src/Findologic/Config/FindologicConfigService.php

```diff
@@ -241,11 +262,22 @@ private function load(string $salesChannelId, string $languageId): array
     private function buildConfig(FinSearchConfigCollection $configs): array
```

Comment:
I may have overseen this last review, but the method is called `buildConfig`, yet it requires `$configs`. I suggest going with the naming of `$options`, since a config contains multiple options. I am still fine with the naming `CONFIG_KEYS`, since it describes all keys inside of a config.